### DISCUSSION TITLE
希望作者能再制作一个Acfun Evolved.......

### DISCUSSION THREAD

**Discussion by Yukihana-Lamy** [the1812/Bilibili-Evolved#3365] [功能建议 / Ideas]
Bilibili Evolved确实不错，希望作者能再制作一个Acfun Evolved，方便各位用户下载A站高清视频.......

**Comment by Geeyun-JY3** [comment]
就视频下载这个需求相关项目还是挺多的，而且应该有不少通用的视频下载项目适配了 a 站的。

**Comment by MC-dusk** [comment]
可以试试这个浏览器扩展（但不能全选多p下载，得自己一个个点）

https://github.com/niuchaobo/acfun-helper

https://gitlab.com/Sokwva/acfun-helper

https://addons.mozilla.org/zh-CN/firefox/addon/acfun%E5%8A%A9%E6%89%8B/

多p下载可以试试you-get、硕鼠、维棠之类的？
